catalog/mapbiomas-public/projects_mapbiomas-public_assets_brazil_lulc_v1.jsonnet

```diff
@@ -0,0 +1,208 @@
+local id = 'projects/mapbiomas-public/assets/brazil/lulc/v1';
+
+local subdir = 'mapbiomas-public';
+
+local version = '1.0';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc_by_4_0;
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  id: id,
+
+  title: 'MapBiomas Land Use and Land Cover - Brazil V' + version,
+
+  version: version,
+
+  description: |||
+    MapBiomas Land Use and Land Cover (LULC) dataset for Brazil is produced annually by the MapBiomas Project using Landsat satellite imagery and machine learning classification techniques.
+    The dataset provides consistent, thematically detailed maps at 30-meter resolution, covering multiple decades and updated each year.
+
+    Each image in the collection contains annual land cover classifications with pixel values representing categorical land cover classes, such as forest, agriculture, pasture, water bodies, and urban areas.
+    The classification legend is standardized and maintained across all years, enabling multi-temporal analyses of land use change, deforestation, reforestation, and other landscape dynamics.
+
+    The classification is based on Landsat 5 (TM), Landsat 7 (ETM+), Landsat 8 and 9 (OLI/TIRS) surface reflectance data, preprocessed and mosaicked to generate cloud-free composites for each year.
+    The classification process includes automated decision trees trained with reference samples, along with manual validation by regional experts.
+
+    Images in this collection follow the naming format:
+
+    ee.ImageCollection('projects/mapbiomas-public/assets/brazil/lulc/v1')
+
+    Each image contains one band named classification, representing the land cover classification for that specific year, version and collection id.
+
+    For more information, classification legend, methodology, and accuracy assessments, visit the [MapBiomas website](https://mapbiomas.org/en).
+
+    It is recommended to consult the classification legend to interpret the class values.
+    Note that MapBiomas provides discrete classification maps, not probabilities.
+    These maps are most appropriate for map transitions, time series analysis, and land policy monitoring applications.
+
+  |||,
+
+  'gee:categories': [
+    'landuse-landcover',
+  ],
+
+  keywords: [
+    'landcover',
```

Comment:
you don't really need the 'landcover' and 'landuse' tags since you specify the landuse-landcover category (which becomes a tag too)